resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -206,7 +185,9 @@ public DnsNameResolverBuilder resolvedAddressTypes(InternetProtocolFamily... res
      *
      * @param resolvedAddressTypes the address types
      * @return {@code this}
+     * @deprecated use {@link #resolvedAddressTypes(ResolvedAddressTypes)} instead
      */
+    @Deprecated
```

Comment:
the duplicate entry removal seems unnecessary here too (could be considered an application error) so I'm not sure we loose much by removing this.